components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/edit.jsp

```diff
@@ -391,6 +406,10 @@
                     //on load adjust the form based on the current settings
                     adjustForm();
                     $("form[name='editAppform']").change(adjustForm);
+
+                    // Set selected encryption algorithm and encryption method.
+                    $('select[name=encryptionAlgorithm]').val('<%=app.getEncryptionAlgorithm()%>');
+                    $('select[name=encryptionMethod]').val('<%=app.getEncryptionMethod()%>');
                 })
```

Comment:
Entries for 'app' reference is being set from request parameters. Thus, these values needs to be properly encoded. Maybe, jquery handles this. Yet, have a check and confirm.

Reply:
Values encoded.